### PR TITLE
kodev: two small fixes

### PR DESCRIPTION
**Pull request by poire-z**
- kodev-run: fix use of $args (see https://github.com/koreader/koreader/pull/3379#issuecomment-346462122)
- kodev-build: build the emulator in debug mode, for consistency with kodev-run (see https://github.com/koreader/koreader/pull/3439#issuecomment-354370470)

### CODE REVIEW

**Comment by Frenzie** [comment]
Note that I gave run a --no-debug flag.

**Comment by poire-z** [comment]
I saw that, but it gets more complicated with kodev-build, which would build non-debug for all except debug for the emulator. Should I just add an undocumented --no-debug to disable this if emulator only?
Or have --debug and --no-debug documented and doing what they mean, just not documenting in the usage what the default is?

**Comment by Frenzie** [comment]
With the changes as in this diff there's no way to get a non-debug build out of the script, which doesn't seem right.

> Or have --debug and --no-debug documented and doing what they mean, just not documenting in the usage what the default is?

What's the objection to documenting it along these lines?

```
        --debug                    debugging symbols (default for emulator)
        --no-debug               no debugging symbols (default for target devices)
```

**Comment by poire-z** [comment]
No objection :) Done.

**Comment by poire-z** [comment]
What's the thing today with luajit luackeck segfaulting ?

File: kodev [hunk 1]
```diff
@@ -207,9 +217,10 @@ ${SUPPORTED_TARGETS}"
             assert_ret_zero $?
             ;;
         *)
-            # As kodev-run (to run the emulator) builds by default a debug build,
-            # let's do that here too.
-            export KODEBUG=1
+            if [ -z "${KODEBUG_NO_DEFAULT+x}" ]; then # no explicite --debug / --no-debug
```
Comment (Frenzie) [inline kodev]: explicit ;-)

File: kodev [hunk 2]
```diff
@@ -207,9 +217,10 @@ ${SUPPORTED_TARGETS}"
...
+                # Builds by default a debug build, as kodev-run does
```
Comment (Frenzie) [inline kodev]: And since I'm policing the comments, "builds a debug build by default, like kodev-run (does)"

**Comment by Frenzie** [comment]
I have no idea. But as long as shellcheck is happy this PR is (soon) a go. :-P

**Comment by TnS-hun** [comment]
Since this commit KOReader no longer reopens the last used document when started with `./kodev run`.

**Comment by poire-z** [comment]
Can you check if you really have the option in File browser menu: `Start with: last file`?
Before this commit, `Start with: file browser` or `Start with: history` wouldn't work: they would always start with last_file. And if you removed last_file from settings.reader.lua, koreader wouldn't start at all.
Well, that's what I observed, I hope I wasn't tricked by some other thing.

**Comment by TnS-hun** [comment]
You're right, it was set to `Start with: file browser`.

I didn't even realize that the path was clickable. :) I tried to click on `KOReader File Browser` to open the menu but that didn't do anything.

**Comment by poire-z** [comment]
Mhhh, that `KOReader File Browser` used to be clickable.

**Comment by Frenzie** [comment]
Side effect of the +? (On phone; can't check.)

**Comment by poire-z** [comment]
Yes indeed, but solution looks simple.

**Comment by Frenzie** [comment]
Oh, I just remembered: the same issue applies to `kodev clean`. (Just to point it out, but feel free to address it if you want.)

**Pull request merged**